data/transactions/logic/eval_test.go

```diff
@@ -1790,20 +1796,34 @@ int 1
 ==
 &&
 `
-	gtxnText := gtxnTextV2 + ` gtxn 0 ExtraProgramPages
+	gtxnTextV4 := gtxnTextV2 + ` gtxn 0 ExtraProgramPages
 int 0
 ==
 &&
 gtxn 1 ExtraProgramPages
 int 2
 ==
 &&
+`
+
+	gtxnTextV5 := gtxnTextV4 + `int 0
```

Comment:
This is missing `txnas`

Reply:
I put `txnas` in the `testTxnProgramTextV5` section I think.